test/QuarkBuilder.t.sol

```diff
@@ -22,8 +22,11 @@ contract QuarkBuilderTest is Test {
     address constant ETH_USD_PRICE_FEED_8453 = 0x71041dddad3595F9CEd3DcCFBe3D1F4b0a16Bb70;
     address constant USDC_1 = 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
     address constant USDC_8453 = 0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913;
-    // Random address for mock of USDC in random unsupported L2 chain
+    address constant USDT_1 = 0xdAC17F958D2ee523a2206206994597C13D831ec7;
+    address constant USDT_8453 = 0xfde4C96c8593536E31F229EA8f37b2ADa2699bb2;
+    // Random address for mock assets in random unsupported chain
     address constant USDC_7777 = 0x8D89c5CaA76592e30e0410B9e68C0f235c62B312;
+    address constant USDT_7777 = 0x8D89c5CaA76592e30e0410B9e68C0f235c62B312;
```

Comment:
We should probably use a different random address than the one we're using for USDC, right?

Might cause issues, even if it's on our unsupported chain

Reply:
Done